packages/drivers/odsp-driver/src/test/mockFetch.ts

```diff
@@ -36,7 +36,7 @@ export async function mockFetchMultiple<T>(
         }
         const cb = responses.shift();
         assert(cb !== undefined, "the end");
-        return cb();
+        return cb() as Promise<fetchModule.Response>;
```

Comment:
Required because TypeScript now knows the API of the function being mocked.

(My assumption is that we don't want to force test code to conjure a fully legit Response, since cb[] is currently typed to {}[])

Reply:
Can you help me understand why the test return type is `Response` from `node-fetch` but the src code deals with `Response` from lib.dom?

Reply:
The signature of the mock must match the signature of the original function being replaced by the [sinon.stub].

The mock with then be invoked via the 'fetch' helper in fetch.ts, which is where the cast occurs.